system/expressionengine/third_party/twitter/mod.twitter.php

```diff
@@ -143,6 +143,39 @@ public function search()
 		return $this->render_tweets($statuses, $prefix, $userprefix);
 	}
 
+	public function getbyid()
+	{
+		// Fetch parameters
+		$this->refresh		= $this->EE->TMPL->fetch_param('twitter_refresh', $this->refresh);
+		$this->limit		= $this->EE->TMPL->fetch_param('limit', '1');
```

Comment:
I don't think we need limit to be a parameter anymore. Lets just remove this parameter, and hardcode `$this->limit = 1` instead. I'm not even sure that's necessary, since Twitter will always return at most 1 tweet for an ID, but it's probably good self-documentation.